src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+// import React from 'react'
+// eslint-disable-next-line no-unused-vars
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] The `App.js` component should be a class-based component. Please, let's fix it
- [ ] Please, let's get rid of commented codes in the codebase. It makes the codebase look messy.

[OPTIONAL]

- [ ] Please, try to fix the eslint error instead of disabling it. This particular error means you have an unused variable. Please, let's fix it.